lab-anthony/lib/greet.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+function greet(name){
```

Comment:
This function looks great! It achieves one of the requirements of the assignment.
However, the assignment has only asked for you to concatenate the word 'hello ' with a name passed into the function. In your function you introduced a default value if a name was was not provided, on line 4. Because this is not following the instructions exactly this is an incorrect submission. In a working environment when a project manager asks for a specific function they don't want any extra code in that function. So keep in mind to read the instructions carefully and follow them exactly :)